src/Mono.Android/Android.Runtime/JNIEnv.cs

```diff
@@ -247,8 +247,13 @@ static void ManualJavaObjectDispose (Java.Lang.Object obj)
 		static void Initialize ()
 		{
 			if (mono_unhandled_exception == null) {
+#if NETCOREAPP
+				var mono_UnhandledException = typeof (Android.Runtime.JNIEnv)
+					.GetMethod ("monodroid_debugger_unhandled_exception", BindingFlags.NonPublic | BindingFlags.Static);
+#else
 				var mono_UnhandledException = typeof (System.Diagnostics.Debugger)
 					.GetMethod ("Mono_UnhandledException", BindingFlags.NonPublic | BindingFlags.Static);
+#endif
```

Comment:
Code convention: `#else` and `#endif` statements should have a comment "describing" the condition they're associated with, e.g.

```csharp
#endif  // NETCOREAPP
```

Reply:
Fixed :)